### ISSUE TITLE
Displaying the statuses of Stimulus controllers in production

### ISSUE THREAD

**Issue by julien-lmnr** [lhapaipai/vite-plugin-symfony#38]
### vite-plugin-symfony version

6.3.4

### vite-bundle version

6.3.4

### your OS, node version, Symfony version, PHP version

Linux, node 18.13.0, symfony 7.0.3, php 8.3.2

### Description

Unlike basic Stimulus integration with Symfony, this bundle displays the various statuses of Stimulus controllers in production. It's great in development, but I think it shouldn't be the case in production.

### How to reproduce

You can see this "issue" on your "playground/stimulus" project.

![stimulus_vite](https://github.com/lhapaipai/vite-plugin-symfony/assets/85691636/10906c1b-785b-4628-a85e-ab322d18b1f7)

**Comment by lhapaipai** [comment]
hi @julien-lmnr,
Vite only gives access to environment variables prefixed with `VITE_`
vite doc: [.env files](https://vitejs.dev/guide/env-and-mode.html#env-files)
therefore this does not give us access to `APP_ENV`. rather than creating an additional environment variable (`VITE_STIMULUS_ENV`), the `vite-plugin-symfony` plugin should be improved so that it reads the `APP_ENV` variable internally.
see stimulus-bridge with webpack : https://github.com/symfony/stimulus-bridge/blob/main/src/index.ts

**Comment by julien-lmnr** [comment]
@lhapaipai

Yes, that's a bit what I did in the meantime in my bootstrap.ts:

```ts
const app = startStimulusApp()

if (process.env.NODE_ENV !== 'development') {
    app.debug = false
}
```

**Comment by lhapaipai** [comment]
fixed in v6.4.1
Have a good day !

**Comment by julien-lmnr** [comment]
@lhapaipai Great! Keep it up.
